include/termcolor/termcolor.hpp

```diff
@@ -39,7 +39,13 @@
 
 
 #include <iostream>
+#ifdef __STRICT_ANSI__  // Required when compiling with
```

Comment:
I forget to mention, BSD tests required a further patch, which was not included here.